test/Tests/src/Semantic/Js_Interop_Spec.enso

```diff
@@ -9,24 +9,24 @@ foreign js debug = """
     debugger;
 
 type My_Type
-    My_Type_Data a b
+    Value a b
 
-    foreign js my_method this = """
+    foreign js my_method self = """
         return this.a + this.b;
```

Comment:
The `self` feels odd - I thought we had a special case for `this` here?

Reply:
I don't think we do? It doesn't work anyway, should be tackled in a separate issue if you think we need it.

Reply:
No special cases for JavaScript outside of `EpbLanguage`, please!